dist/test_data.py

```diff
@@ -79,27 +79,33 @@ def __ge__(self, other):
 # A generic configuration used by various other configurations to define whether that component or
 # similar is enabled or not.
 #
-enable_config = [
+enabled_config_true = [
+    Config('enabled', 'true', r'''
+        Whether or not this is relevant to the workload''',
+        type='boolean'),
+]
+
+enabled_config_false = [
     Config('enabled', 'false', r'''
         Whether or not this is relevant to the workload''',
         type='boolean'),
 ]
 
-stat_config = enable_config
+stat_config = enabled_config_false
```

Comment:
Why would this one be disabled by default ?

Reply:
Well having a limit on cache size could be default able but we'd need to set it to 100. I guess we could leave it on by default but then it may unexpectedly fail a test.

Reply:
I'll leave it as false for now.